idd/Energy+.idd.in

```diff
@@ -40719,6 +40725,9 @@ ZoneHVAC:AirDistributionUnit,
        \key AirTerminal:DualDuct:ConstantVolume
        \key AirTerminal:DualDuct:VAV
        \key AirTerminal:SingleDuct:ConstantVolume:Reheat
+       \key AirTerminal:SingleDuct:ConstantVolume:FourPipeBeam
+       \key AirTerminal:SingleDuct:InletSideMixer
+       \key AirTerminal:SingleDuct:SupplySideMixer
```

Comment:
These two mixers have since been combined and the choice for AirTerminal:SingleDuct:Mixer is already at the bottom of the list, so delete these two lines.